e2e/capl-cluster-flavors/kubeadm-capl-cluster/chainsaw-test.yaml

```diff
@@ -33,10 +35,12 @@ spec:
                 value: ($namespace)
               - name: CLUSTERCTL_CONFIG
                 value: (env('CLUSTERCTL_CONFIG'))
+              - name: KUBERNETES_VERSION
+                value: (env('KUBERNETES_VERSION') || 'v1.29.1')
```

Comment:
I'm wondering if we should set this to a more recent version like 1.31 or even 1.32 in the GHA workflow so it's easier to see what version we're testing with

Reply:
Yeah, thats what I want to do. Currently, the workflow first checks out immutable last tag (0.8.2 in this case) and runs it. Since we can't change it, we have to run v1.29.1 there. I am adding it so that in future when we cut a new tag, we can provide a newer version going forward here using env var.